README.md

```diff
@@ -1,24 +1,110 @@
-# README
+## Budget-app
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+## Table of Contents
 
-Things you may want to cover:
+* [About the Project](#budget_app)
+* [Built With](#built-with)
+* [Getting Started](#getting-started)
+* [Contributing](#🤝-contributing)
+* [Author](#author)
 
-* Ruby version
+# Budget App
+>The best budgeting apps help you understand your income and spending, so you have maximum control over your money.
 
-* System dependencies
+## Built With
 
-* Configuration
+- Ruby
+- Ruby on Rails
+- PostgreSql
+- TailwindCSS
 
-* Database creation
+## Getting Started
 
-* Database initialization
+ [live demo](https://secure-wave-18374.herokuapp.com/)
 
```

Comment:
- [ ] Kindly, make sure that you're using the correct link for the live demo of your project. For now, you're using the wrong link which is not updated with the new features you implemented after the first review. Please, fix this issue.